py-polars/polars/expr/name.py

```diff
@@ -26,7 +26,7 @@ def keep(self) -> Expr:
 
         See Also
         --------
-        alias
+        Expr.alias
```

Comment:
this currently doesn't render, as it's from a different class (here we're in the `.name` namespace)